docs/AboutUs.md

```diff
@@ -58,5 +58,5 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[linkedin](https://www.linkedin.com/in/tayruijie/)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Developer
-* Responsibilities:
+* Role: Documentation
```

Comment:
Perhaps change to something like `Documentation lead` or `Documentation Coordinator`? Would be more descriptive than simply `Documentation`.

Reply:
Yes, that would be good. Resolved!